docs/hardware/lumig.md

```diff
@@ -42,7 +42,7 @@ More information about the AMD MI250x can be found here:
 - [MI250X information on the AMD website][MI250x-amd]
 - [Applications performance and benchmarks][benchmarks]
 
-The LUMI-G nodes CPU is a single 64-cores AMD EPYC 7A53 "Trento" CPU. The cores
+The LUMI-G nodes CPU is a single 64-core AMD EPYC 7A53 "Trento" CPU. The cores
```

Comment:
Not sure removing the plural was correct in this case

Reply:
I think it is – "dual-core CPU" c.f. "dual-cores CPU".

Reply:
Apparently, both are correct but the non-plural is the most common one.